server/data/index.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/first */
```

Comment:
this is more in line with the template project now - lazily creates redis connection which means that the tests don't continually try to create connections with redis